content/articles/announcement-secondary-dns-ip-address-changes.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Secondary DNS IP Address Changes
+excerpt: DNSimple will be updating the IP addresses used for Secondary DNS outbound notifications and zone transfers. The current IPv4 address (50.31.225.92) will be retired beginning [DATE], 2024. Affected customers will be notified via email.
```

Comment:
Remember to set the DATE.

Reply:
Set in https://github.com/dnsimple/dnsimple-support/pull/1258/commits/9f0f9f294517887d36cfa99483ccf0c87282f998